tests/test_post_login.py

```diff
@@ -0,0 +1,22 @@
+import requests
+import pytest
+
+def test_correct_login():
```

Comment:
Take a look at
https://docs.pytest.org/en/latest/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions
and table-driven testing